tests/test_traminformation.py

```diff
@@ -0,0 +1,46 @@
+import json
+import datetime
+import pytest
```

Comment:
Add pytest to dev requirements: https://realpython.com/lessons/production-vs-development-dependencies/